docker-compose.yml

```diff
@@ -30,6 +30,20 @@ services:
         # Hitting :8080/admin/export will force an export to be written to this directory.
         target: /dgraph/export
 
+  # dev uses 1 big redis instance, prod has 1:1 redis per grapl service
+  big-dumb-redis:  # silly name bc temporary
```

Comment:
temporary eh

Reply:
why not keep `big-dumb-redis`?  no objections here

(it's temporary bc prod has one redis instance per service, IIRC, and having just one locally is an intermediate step to that goal)

Reply:
would prefer to rename it something more descriptive like, "shared-redis"